docs/manual/developer_guide.adoc

```diff
@@ -917,3 +917,13 @@ Check out the various `ctest` options to perform specific testing, you can rerun
 Tests are added using the add_test cmake call. Each test should finish with a 0 exit-code in case everything went well and a non-zero if something failed. Output (both stdout and stderr) are collected by ctest and stored in logs or displayed. Make sure you never hard-code a path to any tool when doing testing (or anything really) in the cmake code. Always use configuration to find all the paths and then use the respective variable.
 
 See some of the existing testing code in `cmake/SSGCommon.cmake`.
+
+=== Contribution to infrastructure code
+
+The SSG build and teplating system is mostly written in Python.
+
+==== Python
+
+* The common pattern is to dynamically add the `shared/modules` to the import path. The `ssgcommon` module has many useful utility functions and predefined constants. See scripts at `shared/utils` as an example of this practice.
+* Follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/).
+* Try to keep most of your lines length under 80 characters. Although the 99 character limit is within PEP8 requirements, there is no reason for most lines to be that long.
```

Comment:
isn't pep8 mandating 80 but we changed it to 99 in ssg?

Reply:
Actually not, PEP8 is a living document: https://www.python.org/dev/peps/pep-0008/#maximum-line-length